Doc/library/dis.rst

```diff
@@ -772,8 +772,13 @@ All of the following opcodes use their arguments.
 
 .. opcode:: BUILD_MAP (count)
 
-   Pushes a new dictionary object onto the stack.  The dictionary is pre-sized
-   to hold *count* entries.
+   Pushes a new dictionary object onto the stack.  Pops ``2 * count`` items
+   so that the dictionary holds ``count`` entries:
```

Comment:
`BUILD_TUPLE` and others format the parameter as `*count*` rather than ` ``count`` `.